purchases/src/main/kotlin/com/revenuecat/purchases/common/responses/SubscriptionInfoResponse.kt

```diff
@@ -0,0 +1,54 @@
+package com.revenuecat.purchases.common.responses
+
+import com.revenuecat.purchases.OwnershipType
+import com.revenuecat.purchases.PeriodType
+import com.revenuecat.purchases.Store
+import com.revenuecat.purchases.utils.serializers.ISO8601DateSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.Date
+
+@Serializable
+@SuppressWarnings("LongParameterList")
+internal class SubscriptionInfoResponse(
+    @SerialName("purchase_date") @Serializable(with = ISO8601DateSerializer::class) val purchaseDate: Date,
+    @SerialName("original_purchase_date") @Serializable(with = ISO8601DateSerializer::class) val originalPurchaseDate:
+    Date?,
+    @SerialName("expires_date") @Serializable(with = ISO8601DateSerializer::class) val expiresDate: Date?,
+    @SerialName("store") val store: Store,
+    @SerialName("is_sandbox") val isSandbox: Boolean,
+    @SerialName("unsubscribe_detected_at") @Serializable(with = ISO8601DateSerializer::class) val unsubscribeDetectedAt:
+    Date?,
+    @SerialName(
+        "billing_issues_detected_at",
+    ) @Serializable(with = ISO8601DateSerializer::class) val billingIssuesDetectedAt:
+    Date?,
```

Comment:
Hmm indentation seems weird in some of these?

Reply:
right... detekt is doing this. I will see if I can fix it